Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+pipeline {
+    agent {
```

Comment:
doesn't this need a `logRoatator` setup to remove old builds?

Reply:
I guess.